execution_engine/src/core/runtime/mod.rs

```diff
@@ -1486,7 +1484,8 @@ where
         // Charge just for the amount that particular entry point cost - using gas cost from the
         // isolated runtime might have a recursive costs whenever system contract calls other system
```

Comment:
```suggestion
        // isolated runtime might have recursive costs whenever system contract calls other system
```
(Or "a recursive cost".)